src/posix/platform/config.hpp

```diff
@@ -26,12 +26,12 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef POSIX_PLATFORM_POWER_UPDATER_HPP_
-#define POSIX_PLATFORM_POWER_UPDATER_HPP_
+#ifndef POSIX_PLATFORM_CONFIG_HPP_
+#define POSIX_PLATFORM_CONFIG_HPP_
 
 #include "openthread-posix-config.h"
 
-#if OPENTHREAD_CONFIG_PLATFORM_POWER_CALIBRATION_ENABLE
+#if OPENTHREAD_POSIX_CONFIG_CONFIGURATION_FILE_ENABLE
```

Comment:
Let's add a check for old name of the config (in case some project uses it) and give an error that its replaced with new name?

Reply:
I have added the error message.